extra_data/tests/test_stacking.py

```diff
@@ -155,3 +156,16 @@ def test_stackview_squeeze():
 
     with pytest.raises(np.AxisError):
         sv.squeeze(axis=4)
+
+def test_stack_from_xarray():
+    coords = {'module': np.arange(2),
```

Comment:
I'd like the test to go as far as having a gap in the module numbers (e.g. `[0, 2, 3]`), and verifying that there is data in the expected place and the fill value exposed in the gap.